gui/build.py

```diff
@@ -118,19 +113,48 @@ def parse_args():
 
     bundle = subparsers.add_parser('bundle',
                                    help='bundle the application (freeze)')
+    if IS_WIN:
+        default_python_path = 'C:\\Python27\\Lib'
+        bundle.add_argument('--nsis-path', dest='nsis_path',
+                            default='C:\\NSIS',
+                            help='your NSIS path on the'
+                            ' system(default: %(default)r)')
+    else:
+        default_python_path = '/usr/lib/python2.7'
+
     bundle.add_argument('--python-path', dest='python_path',
-                        default='C:\\Python27',
+                        default=default_python_path,
                         help='your python path on the'
                         ' system(default: %(default)r)')
-    bundle.add_argument('--nsis-path', dest='nsis_path',
-                        default='C:\\NSIS',
-                        help='your NSIS path on the'
-                        ' system(default: %(default)r)')
     bundle.set_defaults(func=do_bundle)
 
     return parser.parse_args()
 
 
+def call_cx_freeze(gui_path, moz_path, python_lib_path):
```

Comment:
I' thinking that gui_path and moz_path may not need to be argument. They are always '.' and '..' (or the equivalent realpath). Maybe you could remove these arguments, and hardcode them like:
```
args.append('--include-path=.;..;%s\site-packages;%s'
```
This would cleanup the code a bit.